runtime-sdk/modules/evm/src/lib.rs

```diff
@@ -59,16 +80,24 @@ pub trait Config: 'static {
     /// The chain ID to supply when a contract requests it. Ethereum-format transactions must use
     /// this chain ID.
     const CHAIN_ID: u64;
+
+    /// Token denomination used as the native EVM token.
+    const TOKEN_DENOMINATION: token::Denomination;
+
+    /// Maps an Ethereum address into an SDK account address.
+    fn map_address(address: primitive_types::H160) -> Address {
+        Address::new(
+            address::ADDRESS_V0_SECP256K1ETH_CONTEXT,
+            address::ADDRESS_V0_VERSION,
+            address.as_ref(),
+        )
+    }
 }
 
 pub struct Module<Cfg: Config> {
     _cfg: std::marker::PhantomData<Cfg>,
 }
 
-/// EVM token pool address.
-pub static ADDRESS_EVM_TOKENS: Lazy<Address> =
-    Lazy::new(|| Address::from_module(MODULE_NAME, "evm-tokens"));
```

Comment:
we used to have this address do an additional layer of accounting to keep the evm itself honest. now we let the evm directly do set_balance on the accounts module.

Reply:
I see that they "apply" in batches. maybe we could check that all the changes balance out. unless the evm can burn. this would take some additional get_balance calls. dunno if it's worth it

Reply:
No those stores shouldn't have a lot of overhead, but there is an extra storage read. But they should be cached anyway as the EVM executor would always first access the account. So these should be cheap?

Reply:
Added a paranoid sanity check.